### ISSUE TITLE
Add a sort function

### ISSUE THREAD

**Issue by samuelmayerhofer** [z-------------/CPod#220]
It would be great if there were a sort function where you could sort episodes from "Oldest to Newest", or from "Longest to Shortest" or vice versa.

**Comment by herzenschein** [comment]
Agreed, this is particularly useful for podcasts that expect the user to listen since the beginning, often the case in language learning podcasts for example.

**Comment by mizzunet** [comment]
Any update on this? For sorting a podcast for listening from the beginning

**Comment by decryptedchaos** [comment]
adding another vote for this.  

this is going to be a deal breaker for me kinda,  its a obvious oversight in the apps design to not have such a feature.  

It is also,  a potential issue that this has been open, and requested for over a year without even the slightest acknowledgement.

**Comment by olafur-palsson** [comment]
I listen to history podcasts, oldest to newest is very important for me.
